custom/enikshay/ucr/data_sources/adherence.json

```diff
@@ -0,0 +1,101 @@
+{
+    "domains": [
+        "enikshay-test",
+        "enikshay-test-test",
+        "enikshay",
+        "enikshay-test-2",
+        "enikshay-test-3",
+        "enikshay-nikshay-migration-test",
+        "enikshay-domain-copy-test",
+        "enikshay-aks-audit",
+        "np-migration-3"
+    ],
+    "config": {
+        "referenced_doc_type": "CommCareCase",
+        "engine_id": "ucr",
+        "backend_id": "ES",
```

Comment:
@sravfeyn what are the benefits you see from using ES here? It looks like you're just doing some basic filtering for now.